Code/Engine/Foundation/Reflection/Implementation/PropertyAttributes.cpp

```diff
@@ -144,6 +144,22 @@ EZ_BEGIN_DYNAMIC_REFLECTED_TYPE(ezDefaultValueAttribute, 1, ezRTTIDefaultAllocat
 }
 EZ_END_DYNAMIC_REFLECTED_TYPE;
 
+EZ_BEGIN_DYNAMIC_REFLECTED_TYPE(ezSliderAttribute, 1, ezRTTIDefaultAllocator<ezSliderAttribute>)
```

Comment:
No I asked for this. The clamp attribute only does it for limited ranges automatically, it doesn't add a slider unconditionally, which is what we want here.
Also later I would like to add a completely new widget type for when this attribute exists, so that we can use dedicated images for things like the color temperature.

Reply:
This came out of a convocation with @jankrassnigg due to ezClampValueAttribute having a hard coded upper limit

Reply:
Discussed offline with Jan we leave it like this for now, so nothing to do on your end.